interface/modules/zend_modules/module/Carecoordination/view/layout/mapper.phtml

```diff
@@ -21,16 +21,16 @@
                         ?>
 
                         <?php echo $this->headScript()
-                        ->prependFile($this->basePath() . '/js/carecoordination/jquery.treeview.js')
-                        ->prependFile($this->basePath() . '/js/carecoordination/jquery.cookie.js')
-                        ->prependFile($this->basePath() . '/js/carecoordination/virtualpaginate.js')
+                        ->prependFile($this->basePath() . '/js/lib/jquery.treeview-1.4.1/jquery.treeview.js')
+                        ->prependFile($this->basePath() . '/js/lib/jquery.treeview-1.4.1/lib/jquery.cookie.js')
+                        ->prependFile($this->basePath() . '/js/lib/virtualpaginate.js')
                                     ->prependFile($this->basePath() . '/js/carecoordination/ui/jquery.ui.tabs.js')
                                     ->prependFile($this->basePath() . '/js/carecoordination/ui/jquery.ui.sortable.js')
-                                    ->prependFile($this->basePath() . '/js/carecoordination/ui/jquery.ui.mouse.js')
+                                    ->prependFile($this->basePath() . '/js/lib/jquery.ui.mouse.js')
                                     ->prependFile($this->basePath() . '/js/carecoordination/ui/jquery.ui.widget.js')
-                                    ->prependFile($this->basePath() . '/js/carecoordination/ui/jquery.ui.core.js')
-                                    ->prependFile($this->basePath() . '/js/carecoordination/jquery-1.8.3.js')
-                                    ->prependFile($this->basePath() . '/js/carecoordination/jquery.js')
+                                    ->prependFile($this->basePath() . '/js/lib/jquery.ui.core.js')
+                                    ->prependFile($this->basePath() . '/js/lib/jquery-1.8.3.js')
```

Comment:
Testing: We should at least point to /public/assets in this file so we can get rid of the excess files.